src/mutate/index.js

```diff
@@ -63,44 +53,33 @@ const decorateEditBase = args => {
                 (args.params && args.params.id) ||
                 props.match.params.id
             },
-            fetchPolicy: "cache-and-network",
-            refetchQueries: [
-              {
-                query: gqlFetchList(mutationVars.queryName, args.fields)
-              }
-            ]
+            fetchPolicy: "cache-and-network"
           };
         },
         props: props => ({
-          formData: processFormData(props.data[mutationVars.detailQueryName]),
           [dataKey || `data`]: props.data[mutationVars.detailQueryName],
           loading: props.data.loading,
           apolloInternalError: props.data.error
         })
       }
     ),
+    branch(props => !props[dataKey || `data`], renderNothing),
```

Comment:
Feels wrong to have this here, as it basically locks us into a loading state of
`null` for for every `decorateEdit` component.  Any way to avoid it so we can
control what happens when things are loading lower down the tree.

Reply:
@pckilgore it is wrong lol #hackgang

Reply:
im going to figure out how to remove it shouldn't be that difficult